### PR TITLE
feat: UUID v5 command

### PR DESCRIPTION
**Pull request by LunaticMuch**
## Description

Adds UUID v5 options as defined in [RFC 4122](https://datatracker.ietf.org/doc/html/rfc4122).
To support the new command, an additional option for `NAMESPACE` is now required.

Addresses #5826 

## Screencast

Screenshot 2023-04-08 at 10 11 57
Screenshot 2023-04-08 at 10 11 38

## Checklist

- [X] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [X] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [X] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [X] I checked that files in the `assets` folder are used by the extension itself
- [X] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
@LunaticMuch are you still around?

**Comment by LunaticMuch** [comment]
> @LunaticMuch are you still around?

Indeed I am. Sorry I have not seen comments. Give me 24 hours I'll be on them 🙄

File: extensions/uuid-generator/src/generateV5.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  } catch (e) {
+    switch ((e as { message: string }).message) {
+      case "INVALID_NAMESPACE":
+        await showHUD(`❌ Invalid namespace definition. Namespace must be a valid UUID.`);
+        break;
+      case "INVALID_NUMBER":
+        await showHUD(`❌ Invalid number provided. Try an actual number.`);
+        break;
+    }
+  }
```
[the remark this answers is not in the document]
Comment (LunaticMuch) [reply extensions/uuid-generator/src/generateV5.tsx]: Agree. Did the change and, for consistency, I applied it also do V4. In this way any failure throws a toast. The successful execution still uses HUD and closes raycast

File: extensions/uuid-generator/package.json [hunk 2]
```diff
@@ -23,20 +24,49 @@
...
+    },
+    {
+      "name": "uuidNamespace",
+      "type": "textfield",
+      "required": false,
+      "default": "",
+      "title": "UUID v5 Namespace",
+      "label": "Namespace",
+      "description": "The namespace for UUID v5"
+    }
```
[the remark this answers is not in the document]
Comment (LunaticMuch) [reply extensions/uuid-generator/package.json]: Fixed

**Review by thomaslombart — APPROVED** [review]
🚀

**Comment by LunaticMuch** [comment]
> Oh, just saw that there were no entry in the `CHANGELOG.md` file. Could you add one @LunaticMuch?

Done 🥲

**Pull request merged**